jessica_hart/assignments/Flask/dojoSurvey/server.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask, render_template, request, redirect
+
+app = Flask(__name__);
+
+@app.route('/')
+def index():
+    return render_template("index.html")    # Render the template and return it
+
+@app.route('/result', methods = ['POST', 'GET'])
+def result():
```

Comment:
You are accepting post and get, but don't do anything for a get. If you're not expecting it, no need to add it.